site/src/atomics/typography.md

```diff
@@ -114,3 +116,21 @@ Appending with `-tablet` to a text-align class will make that class applicable t
 ```html
 <p class="text-align-center-tablet">Text is centered on tablet size and above.</p>
 ```
+
+## Text wrapping
+
+The `.text-wrap-pretty` atomic can be used to prevent [orphans](https://en.wikipedia.org/wiki/Widows_and_orphans), or situations where a line breaks and just one small word ends up on the next line. When the `.text-wrap-pretty` atomic is applied, the browser will instead try to cut the line so the break looks a little more balanced.
+
+If your browser supports pretty text-wrapping, resize your browser window and notice the differences between how these two paragraphs break.
+
+```html
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut.</p>
+<p class="text-wrap-pretty">
```

Comment:
Super nit: can you add `margin-top-xxs` to the second paragraph so they're separated a little please?

Reply:
Added!